lib/studies/reply/mail_card_preview.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+import 'package:gallery/layout/adaptive.dart';
+import 'package:gallery/studies/reply/colors.dart';
+import 'package:gallery/studies/reply/mail_view_page.dart';
+
+const _assetsPackage = 'flutter_gallery_assets';
```

Comment:
I think this const already exist somewhere else. You might be able to use that instance instead

Reply:
It is, but its a private const, maybe this is something we can add to gallery_options.dart and move all the demos to use this.